tsconfig.json

```diff
@@ -1,13 +1,21 @@
 {
   "compilerOptions": {
     "outDir": "./dist/",
```

Comment:
I remember you saying there was a problem with putting them in the same folder, but I don't understand what's the problem yet.